contracts/libraries/tableland/VehicleTableStorage.sol

```diff
@@ -0,0 +1,22 @@
+//SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.13;
+
+/// TODO Documentation
+library VehicleTableStorage {
+    bytes32 internal constant VEHICLE_TABLE_STORAGE_SLOT =
+        keccak256("DIMORegistry.vehicleTable.storage");
+
+    struct Storage {
+        // A mapping that holds `manufacturer ID` and its `tableId`
+        mapping(uint256 => uint256) tables;
```

Comment:
Remember that we can do 
```suggestion
        mapping(uint256 manufacturerId => uint256 tableId) tables;
```
now.

Reply:
Not is this solidity version. Not sure if we can have a different version in the modules. Let me check.

Reply:
Oh whoops.